ledger/src/shredder.rs

```diff
@@ -93,6 +93,7 @@ impl Shredder {
                 self.version,
                 self.reference_tick,
                 is_last_in_slot,
+                None, // chained_merkle_root
```

Comment:
Following commits will do the plumbing here.

Reply:
for the first FEC set will we chain the merkle root from the last FEC set of the parent block?

Reply:
@AshwinSekar no real objection to do so (aside from slightly more code to look that up from blockstore),
but can you clarify the motivation to do so?

Reply:
essentially it gives us a unique identifier for a fork. this is useful for comparison if we add the merkle root to votes.

imagine we have some unplayed slot S in our blockstore and S gets confirmed through votes. we can easily compare S to the merkle root in the votes to verify that we have the correct version of S.

if we also chain the merkle root to the parent block, the same comparison will tell us if we have the correct version of S - 1, S -2, and such ancestors. 

in the event that we have parent block P and child block C where the chaining of the first FEC set of C does not match the last FEC set of P, we can conclude that we have incompatible versions of P and C. Depending on which merkle root the votes for C or its descendants contain, we can reconcile the situation by dumping and repairing P or C appropriately.